.gitignore

```diff
@@ -422,3 +422,4 @@ DerivedData/
 /libs
 
 # End of https://www.gitignore.io/api/xcode
+/SourceS/config.h
```

Comment:
The output file should be in the build folder, so this file shouldn't be needed in the ingnore

Reply:
The build folder is not part of includes.
I'm currently trying to migrate to CMake, so hopefully this won't be necessary.

Reply:
It is :)
https://github.com/diasurgical/devilutionX/blob/master/CMakeLists.txt#L214

Reply:
Yeah it's just not in the Makefile